### DISCUSSION TITLE
IMAP + EMAIL Channel configuration with Amazon SES

### DISCUSSION THREAD

**Discussion by ThalesGnimavo** [chatwoot/chatwoot#4706] [Self hosted] [answered]
Hello,
We have successfully configured the smtp with Amazon SES and it works fine. Users are able to sign up, received notifications etc.

But I unable to setup EMAIL CHANNEL and iMAP. It generate the forwrded email ae5e4ed26b9fef083bc81519c80c139a@email.tpxxx.com but no mail is received. Also it is impossible to add IMAP account. 

Should I install postfix first on same chatwoot server? any tutorial step by step from RAILS or mail server installation ?

Thanks

**Comment by ThalesGnimavo** [comment]
We have a mailbox with IMAP access on another server and all mails sent to that mailbox should be received on chatwoot via email channel.

**Comment by sojan-official** [comment]
@justethales  could you give more detail on " it is impossible to add an IMAP account." are you facing any errors?

**Comment by ThalesGnimavo** [comment]
<img width="505" alt="Capture d’écran 2022-05-23 à 20 30 34" src="https://user-images.githubusercontent.com/43754506/169884027-8c43db22-77c6-44a1-8771-38127a9ada52.png">

**Comment by ThalesGnimavo** [comment, marked as answer]
Sorry. it worked fine now. I forget to add imap port 993. I put 465 instead  of 993. Thanks.

Is there any possibility to compose new mail from Chatwoot  email channel?

**Reply by sojan-official** [reply]
Thanks for reporting back. 

for outbound conversations ref: https://github.com/chatwoot/docs/issues/249
